src/custom_handler.rs

```diff
@@ -4,22 +4,28 @@ use crate::app::CosmosRouter;
 use crate::error::{bail, AnyResult};
 use crate::{AppResponse, Module};
 use cosmwasm_std::{Addr, Api, Binary, BlockInfo, Empty, Querier, Storage};
-use derivative::Derivative;
 use std::cell::{Ref, RefCell};
 use std::ops::Deref;
 use std::rc::Rc;
 
 /// A cache for messages and queries processes by the custom module.
-#[derive(Derivative)]
-#[derivative(Default(bound = "", new = "true"), Clone(bound = ""))]
-pub struct CachingCustomHandlerState<ExecC, QueryC> {
+#[derive(Default, Clone)]
+pub struct CachingCustomHandlerState<ExecC, QueryC>
+where
```

Comment:
Since the `new` method is not anymore generated with the `derivative` crate I think it should be manually added in case anyone uses that.
I don't think this is a case, but as this type is public I we should not break API here.